mev/mev/settings_production.py

```diff
@@ -20,13 +21,37 @@
 # START logging settings
 ###############################################################################
 
-# setup some logging options for production:
-production_config_dict = copy.deepcopy(log_config.base_logging_config_dict)
-
-# make changes to the default logging dict here:
+# Change the LOGLEVEL env variable if you want logging
+# different than INFO:
+LOGLEVEL = os.environ.get('LOGLEVEL', 'info').upper()
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'handlers': {
+        'console': {
+            'class': 'logging.StreamHandler',
+        },
+    },
+    'root': {
+        'handlers': ['console'],
+        'level':  LOGLEVEL,
+    },
+}
```

Comment:
This section appears to be duplicated in dev settings, so it would make more sense to put it in `base_settings`.

Reply:
Agreed. I initially had some differences in `LOGGING` between the dev and production modules, but this can be cleaned up

Reply:
So, this block can now be removed, right?

Reply:
Changed, but forgot to hit save for last commit....